CentOS-7/cms/wordpress-nginx.yml

```diff
@@ -0,0 +1,99 @@
+      }
+
+  - path: /etc/nginx/conf.d/default.conf
+    content: |
+      server {
+        listen 80 default_server;
+        listen [::]:80 default_server ipv6only=on;
+        root /var/www/html;
+        index index.php index.html index.htm;
+        server_name localhost;
+        location / {
+            # First attempt to serve request as file, then
+            # as directory, then fall back to displaying a 404.
+            try_files $uri $uri/ =404;
+            # Uncomment to enable naxsi on this location
+            # include /etc/nginx/naxsi.rules
+        }
+        error_page 404 /404.html;
+        error_page 500 502 503 504 /50x.html;
+        location = /50x.html {
+            root /usr/share/nginx/html;
+        }
+        location ~ \.php$ {
+            try_files $uri =404;
+            fastcgi_split_path_info ^(.+\.php)(/.+)$;
+            fastcgi_pass unix:/var/run/php-fpm/php-fpm.sock;
+            fastcgi_index index.php;
+            fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+            include fastcgi_params;
+        }
+      }
+  - path: /var/www/html/info.php
+    content: |
+      <?php
+      phpinfo();
+      ?>
+runcmd:
+  - yum -y install epel-release
+  - yum -y install unzip nginx php-fpm php-mysql mariadb-server mariadb
+  - wget https://wordpress.org/latest.zip -O /tmp/wordpress.zip
+  - unzip /tmp/wordpress.zip -d /tmp/
+  - cp /tmp/wordpress/wp-config-sample.php /tmp/wordpress/wp-config.php
+  - ROOTMYSQLPASS=`dd if=/dev/urandom bs=1 count=12 2>/dev/null | base64 -w 0 | rev | cut -b 2- | rev``
+  - WPMYSQLPASS=`dd if=/dev/urandom bs=1 count=12 2>/dev/null | base64 -w 0 | rev | cut -b 2- | rev``
+  - echo "Root MySQL Password $ROOTMYSQLPASS" > /root/passwords.txt
+  - echo "Wordpress MySQL Password $WPMYSQLPASS" >> /root/passwords.txt
+  - sed -ie "s/database_name_here/wordpress/" /tmp/wordpress/wp-config.php
+  - sed -ie "s/username_here/wordpress/" /tmp/wordpress/wp-config.php
+  - sed -ie "s/password_here/$WPMYSQLPASS/" /tmp/wordpress/wp-config.php
+  - for i in `seq 1 8`; do wp_salt=$(</dev/urandom tr -dc 'a-zA-Z0-9!@#$%^&*()\-_ []{}<>~`+=,.;:/?|' | head -c 64 | sed -e 's/[\/&]/\\&/g'); sed -i "0,/put your unique phrase here/s/put your unique phrase here/$wp_salt/" /tmp/wordpress/wp-config.php; done
+  - systemctl enable mariadb
+  - systemctl start mariadb
+  - /usr/bin/mysqladmin -u root -h localhost create wordpress
+  - /usr/bin/mysqladmin -u root -h localhost password $ROOTMYSQLPASS
+  - /usr/bin/mysql -uroot -p$ROOTMYSQLPASS -e "CREATE USER wordpress@localhost IDENTIFIED BY '"$WPMYSQLPASS"'"
+  - /usr/bin/mysql -uroot -p$ROOTMYSQLPASS -e "GRANT ALL PRIVILEGES ON wordpress.* TO wordpress@localhost"
+  - mkdir -p /var/www/html
+  - sed -ie "s/;cgi.fix_pathinfo=1/cgi.fix_pathinfo=0/" /etc/php.ini
+  - sed -ie "s|listen = 127.0.0.1:9000|listen = /var/run/php-fpm/php-fpm.sock|" /etc/php-fpm.d/www.conf
+  - cp -Rf /tmp/wordpress/* /var/www/html/.
```

Comment:
I think you need fix ownership after moving them into here. It's all owned by root as it is.